communications.md

```diff
@@ -1,68 +1,89 @@
+Minimize Notifications: Nowadays we have way too many channels of receiving information, by choice or otherwise. Instead of striving to achieve "Notification Zero", try to be as close as possible to "Everything Zero". This means diligently clear or defer any information that is not work-related. Some platforms like Slack offer one simple toggle to enter the "Do Not Disturb" mode. Use it to minimize distractions, so you can find what you need to focus on, faster.
 
-As for emails, you may do the following:
+Opt-out: Leave channels that you are no longer interested in or obligated to.
 
-- Pin/star items that need future attention. Archive them as soon as they're done, or no longer needed.
-- Optionally snooze items that need future attention. Archive them as soon as they're done, or no longer needed.
-- Archive the rest.
-
-## Asynchronous by Default
-
-Whenever you want to communicate with others, default to an asynchronous way. By doing so, you respect your coworker's time and the fact that they may be in the "zone" and should not be interrupted if you are at all considerate about others and their productivity.
+Asynchronous Communication by Default: Whenever you want to communicate with others, default to an asynchronous way. By doing so, you respect your coworker's time and their productivity and the fact that they may be in the "zone" and should not be interrupted. 
 
 When necessary, elevate your message's attention-seeking priority in such order:
 
-1. Send a well-written message in a relevant Slack channel, or GitHub issues, etc.
+1. Send a well-written message in a relevant Slack channel
 2. If this needs a certain person's or group's attention, ping them right below or start a thread and ping them there.
 3. In a rare "meltdown" level of urgency, yell out in any synchronous manner and get people's attention.
 
-Asynchronous communication has an innate chaotic nature. It should be every participant's, at least initiator's obligation to organize and summarize afterward and write it in well-formatted documentation as a part of our internal knowledge base. At a bare minimum, copy the transcript of relevant discussions into a place for future reference or refinement to be retained in the knowledge base.
-
-## Text by Default
-
-Before you decide to have a face-to-face or voice/video conference, take some time to organize your thoughts, and write them down. Revise it to be as concise as possible. If it's not urgently blocking all of your next tasks, asynchronously send this message.
+Documentation - Consolidation: Asynchronous communication can become chaotic so it should be every participant's, at least the initiator's obligation to organize and summarize afterwards and write it in well-formatted documentation as a part of our internal knowledge base. At a bare minimum, copy the transcript of relevant discussions into a place for future reference or refinement to be retained in the knowledge base.
```

Comment:
~afterwards~ afterward

Reply:
also ping @Goondrious @dshparyk on possible (semi-)automation of this process. with one detail for @maluhoss to incorporate into this part:

1. Use Slack thread whenever possible
2. Bot script to get thread message ID as input, fetches all thread messages through Slack API
3. Post it to a transcript repository (let's say on GitHub), as an issue, through GitHub API
4. (optionally badshitcrazywild thought) Utilize a NLP model to assist the summarization of the transcripts.
5. ???
6. ~Profit~ semi-assisted documentation reference.